extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/output/AzureOutputConfig.java

```diff
@@ -64,14 +57,12 @@ public class AzureOutputConfig
   public AzureOutputConfig(
       @JsonProperty(value = "container", required = true) String container,
       @JsonProperty(value = "prefix", required = true) String prefix,
-      @JsonProperty(value = "tempDir", required = true) File tempDir,
```

Comment:
This is created from the runtime properties, and should not be passed between workers. TmpDirectories _should_ not have been passed anywhere, since they are "per peon" properties, and passing the value doesn't make sense.

The only changed class that should have been passed between peons is the exportStorageProvider classes, and these do not contain the tmpDirectory values.